### PR TITLE
feat(Portal): Remove mention of Portal from documentation part 2 

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [RUNTIME-304](https://bonitasoft.atlassian.net/browse/RUNTIME-304)

### CODE REVIEW

**Comment by DumitruCorini** [comment]
Need to modify platform with runtime in most of the cases

**Comment by DumitruCorini** [comment]
- Need to replace bpm-applications with applications in file names
- Duplicated line at start of files with {description}
- log into bonita portal or bonita platform with bonita runtime

**Comment by DumitruCorini** [comment]
I added the changes that were required. Can go to pr in review.

File: modules/ROOT/pages/configurable-archive.adoc [hunk 1]
```diff
@@ -11,10 +11,10 @@ For Enterprise and Performance editions only.
...
 It enables you to specify for each FlowNode type whether or not to archive values when the flow node is changed. This determines what information
-(for example archived instances of processes and executed tasks) is available to an administrator in Bonita Portal.
+(for example archived instances of processes and executed tasks) is available to an administrator in Bonita Administrator Application.
 
 Deactivating archives has a positive effect on overall performance of the system as it reduces the number of entries inserted in the database. It has also the benefit of reducing the disk space used by the database.
-However, it prevents users from keeping a history of past activity. The most visible impact of this loss of information is through the Bonita Portal. Completed tasks will not be visible.
+However, it prevents users from keeping a history of past activity. The most visible impact of this loss of information is through the Bonita Applications. Completed tasks will not be visible.
 
 In general we recommended that you set all attributes to *true* or all to *false*. However, here are some details on some settings that you can change if users have a specific use case:
```
Comment (vbotnari12) [inline modules/ROOT/pages/configurable-archive.adoc]: In general we recommended that you set all attributes to *true* or all to *false*. => In general, we recommend you to set all attributes to *true* or all to *false*.

File: modules/ROOT/pages/configure-a-test-organization.adoc [hunk 2]
```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
...
 
 To upload the organization data:
 
-. Choose Publish... from the Organization menu.
-. Click the organization to be uploaded, and then click Next.
-. Specify the username and password of the user that will be used to log in to Bonita Portal when a process is started from Bonita Studio. This user must be in the organization you have selected.
-. Click Publish.
+. Choose Deploy from the Organization menu.
+. Click the organization to be uploaded.
+. Specify the username of the user that will be used to log into the Bonita Runtime when a process is started from Bonita Studio. This user must be in the organization you have selected.
```
Comment (vbotnari12) [inline modules/ROOT/pages/configure-a-test-organization.adoc]: This user must be in the organization you have selected. => This user must be from the organization you have selected.

File: modules/ROOT/pages/configure-a-test-organization.adoc [hunk 3]
```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
...
+. Click Deploy.
 
 After it has been uploaded, you can xref:organization-in-bonita-applications-overview.adoc[modify the organization in Bonita Portal].
```
Comment (vbotnari12) [inline modules/ROOT/pages/configure-a-test-organization.adoc]: modify the organization in Bonita Portal]. => modify the organization in Bonita Applications].

File: modules/ROOT/pages/create-application.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 The last step of this Getting Started tutorial is to create an application.
 
-To create and deploy a first application in your local environment, use the *xref:applicationCreation.adoc[Application descriptor]*. This editor defines the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help deploy the application directly from Bonita Studio.
+To create and deploy a first application in your local environment, use the *xref:applicationCreation.adoc[Application descriptor]*. This editor defines the skeleton of an application, as it contains references to resources installed in the Bonita Runtime and used by a given application. It will also help deploy the application directly from Bonita Studio.
```
Comment (vbotnari12) [inline modules/ROOT/pages/create-application.adoc]: To create and deploy a first application in your local environment, => To create and deploy an application in your local environment,

If the deploiement process is the same for any application.

File: modules/ROOT/pages/create-application.adoc [hunk 5]
```diff
@@ -42,7 +42,7 @@ image:images/getting-started-tutorial/create-application/create-application.gif[
 The look & feel can be fully customized by modifying the xref:layouts.adoc[layout] and adding a custom xref:customize-living-application-theme.adoc[theme].
 ====
 
-The application is ready to be used. Now you need to deploy it in Bonita Portal:
+The application is ready to be used. Now you need to deploy it in Bonita Applications:
```
Comment (vbotnari12) [inline modules/ROOT/pages/create-application.adoc]: The application is ready to be used. Now you need to deploy it in Bonita Applications: => The application is ready to be deploied in Bonita Applications. Once done, your application is ready to be used. 

from user perspective "ready to be used" means no other activities are to be done, but we still have the deploiement to do, so, for me the application is not ready to be used. Maybe ready to be consumed, but the use is for final users.

File: modules/ROOT/pages/create-application.adoc [hunk 6]
```diff
@@ -58,14 +58,11 @@ Congratulations! You have successfully created your first process and your first
 [NOTE]
 ====
 
-Verify that the application has been created in Bonita Portal.
+Verify that the application is available.
```
Comment (vbotnari12) [inline modules/ROOT/pages/create-application.adoc]: Verify that the application is available. => 
1. Confirm that the application is available.
2. Make sure that the application is available.

- verify is often misleading. It gives the impression that there may be a tiny possibility for the application not to be there...

File: modules/ROOT/pages/create-application.adoc [hunk 7]
```diff
@@ -58,14 +58,11 @@ Congratulations! You have successfully created your first process and your first
...
 
-. Click on the Portal icon image:images/getting-started-tutorial/create-application/portal-icon.png[Portal icon] in the Bonita Studio coolbar.
-. In the upper right corner of the window, click on the *User* drop down menu
-. Select *Administrator*
-. Click on the *Applications* tab
+. Click on the Applications icon image:images/getting-started-tutorial/create-application/applications-icon.png[Applications icon] in the Bonita Studio coolbar.
+. You will see the application _Claims_ that you have just deployed listed in Bonita Application Directory. If you click on the URL, _My Dashboard_ page will be displayed in your brower.
```
Comment (vbotnari12) [inline modules/ROOT/pages/create-application.adoc]: brower ? or browser?

File: modules/ROOT/pages/create-or-modify-a-page.adoc [hunk 8]
```diff
@@ -120,12 +120,12 @@ The preview displays the element as it would be displayed on the selected type o
 
 == Export an element
 
-You can export a page or a layout to deploy it in Bonita Portal as a custom page. +
+You can export a page or a layout to deploy it in Bonita Admin Application as a custom page. +
 You can export any element to import it into another UI Designer. +
 
 To export an element, click the *_Export_* button image:images/images-6_0/pb-export.png[Export button] on the Page editor or the UI Designer home page. A zip file is downloaded to your computer. It contains a Bonita custom page, which also suitable for import into another UI Designer. +
```
Comment (vbotnari12) [inline modules/ROOT/pages/create-or-modify-a-page.adoc]: which also suitable =>  which is also suitable

File: modules/ROOT/pages/csrf-security.adoc [hunk 9]
```diff
@@ -1,7 +1,7 @@
 = CSRF security
 :description: Bonita REST API is secured against Cross-Site Request Forgery (CSRF) attacks.
 
-Bonita REST API is secured against Cross-Site Request Forgery (CSRF) attacks.
+{description}
 
 CSRF (also known as XSRF and Sea-Surf), is an example of a malicious attack on a trusted website.
 While the user is authenticated, by being logged on to a trusted website, malicious requests are transmitted without the user knowing, or giving their consent.
```
Comment (vbotnari12) [inline modules/ROOT/pages/csrf-security.adoc]: by being logged on to a trusted website => by being logged on a trusted website

I may be wrong, but it was difficult to read the phrase at once.

File: modules/ROOT/pages/customize-layouts.adoc [hunk 10]
```diff
@@ -23,14 +23,15 @@ Prerequisites to customize a layout:
...
+* Export the layout from Bonita Administrator Application
 * Import the layout into the UI Designer
 * Modify the layout
 * Apply the customized layout onto a living application.
 
-== Export a layout from Bonita Portal
+== Export a layout from Bonita Administrator Application
 
-. Open Bonita Portal, and log in with username and password of someone mapped to the Administrator profile.
+. Log in with someone mapped to the Administrator profile.
```
Comment (vbotnari12) [inline modules/ROOT/pages/customize-layouts.adoc]: Log in with someone mapped to the Administrator profile. => Log in with a users having the Administrator profile.

its not a date to go with someone 👯

File: modules/ROOT/pages/customize-layouts.adoc [hunk 11]
```diff
@@ -70,13 +71,13 @@ Once your changes are made, save the new layout using a new name and then export
 . Click on *Save*.
 . Click on the *Export* button.
 
-== Import the _myCustomLayout_ into the portal
+== Import the _myCustomLayout_ into the Bonita Administrator Application
 
-. Open Bonita Portal, and again log in with username and password of someone mapped to the Administrator profile.
+. Log in with someone mapped to the Administrator profile.
```
Comment (vbotnari12) [inline modules/ROOT/pages/customize-layouts.adoc]: Log in with someone mapped to the Administrator profile. => Log in with a users having the Administrator profile.

its not a date to go with someone 👯

File: modules/ROOT/pages/configurable-archive.adoc (hunk 1 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configurable-archive.adoc]: Added

File: modules/ROOT/pages/configure-a-test-organization.adoc (hunk 2 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configure-a-test-organization.adoc]: Added

File: modules/ROOT/pages/configure-a-test-organization.adoc (hunk 3 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configure-a-test-organization.adoc]: Added

File: modules/ROOT/pages/create-application.adoc (hunk 4 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-application.adoc]: Added

File: modules/ROOT/pages/create-application.adoc (hunk 5 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-application.adoc]: Added

File: modules/ROOT/pages/create-application.adoc (hunk 6 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-application.adoc]: Added

File: modules/ROOT/pages/create-application.adoc (hunk 7 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-application.adoc]: image

File: modules/ROOT/pages/create-or-modify-a-page.adoc (hunk 8 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-or-modify-a-page.adoc]: Added

File: modules/ROOT/pages/csrf-security.adoc (hunk 9 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/csrf-security.adoc]: I simplified the phrase to "While the user is authenticated through a trusted website, malicious requests are transmitted without the user giving his consent or even knowing."

File: modules/ROOT/pages/customize-layouts.adoc (hunk 10 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/customize-layouts.adoc]: A lot of people do pair programming, they might log in in pair. Don't discriminate ❤️

File: modules/ROOT/pages/create-application.adoc (hunk 7 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/create-application.adoc]: I modified it to browser

File: modules/ROOT/pages/customize-layouts.adoc (hunk 10 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/customize-layouts.adoc]: I added it

File: modules/ROOT/pages/customize-layouts.adoc (hunk 11 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/customize-layouts.adoc]: Modified this one as well

File: modules/ROOT/pages/engine-architecture-overview.adoc [hunk 12]
```diff
@@ -91,7 +88,7 @@ BPM services are related to BPM activities.
 |  |
 
 | Description:
-| Manage categories of processes. A category is a way to classify processes. Categories are handled manually by administrators in Bonita Portal and are visible to portal users.
+| Manage categories of processes. A category is a way to classify processes. Categories are handled manually by in the Bonita Administrator Application and are visible to all users.
```
Comment (vbotnari12) [inline modules/ROOT/pages/engine-architecture-overview.adoc]: Categories are handled manually by in the Bonita => Categories are handled manually by administrators in the Bonita

File: modules/ROOT/pages/execution-sequence-states-and-transactions.adoc [hunk 13]
```diff
@@ -1,11 +1,9 @@
 = BPM process / task execution sequence, states, transitions
 :description: Understand the sequence of actions performed when a process instance is created and when a task is performed by a user
 
-Understand the sequence of actions performed when a process instance is created and when a task is performed by a user
+{description}
 
-This page also shows the interactions between the Bonita Portal and Bonita Engine.
-
-In the sequences described in this page, some of the actions apply only to *Teamwork*, *Efficiency*, *Performance*, or *Enterprise* editions, and are marked "Subscription only".
+This page also shows the interactions between the Bonita Applications and Bonita Engine.
```
Comment (vbotnari12) [inline modules/ROOT/pages/execution-sequence-states-and-transactions.adoc]: between the Bonita Applications => between Bonita Applications

File: modules/ROOT/pages/configurable-archive.adoc [hunk 14]
```diff
@@ -27,11 +27,11 @@ In general we recommended that you set all attributes to *true* or all to *false
 * To archive a loop activity each time it changes state, set `bonita.tenant.archive.SLoopActivityInstance` to true.
 * To archive a multi-instances activity each time it changes state, set `bonita.tenant.archive.SMultiInstanceActivityInstance` to true.
 * To archive a call activity each time it changes state, set `bonita.tenant.archive.SCallActivityInstance` to true.
-* To archive a gateway each time it changes state, set `bonita.tenant.archive.SGatewayInstance` to true. This information is not available in Bonita Portal.
+* To archive a gateway each time it changes state, set `bonita.tenant.archive.SGatewayInstance` to true. This information is not available in the Bonita Applications.
```
Comment (vbotnari12) [inline modules/ROOT/pages/configurable-archive.adoc]: not available in the Bonita Applications. => not available in Bonita Applications.

To be coherent with line 33.

File: modules/ROOT/pages/configure-a-test-organization.adoc [hunk 15]
```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
 == Export and import an organization
 
 You can export an organization definition from Bonita Studio. The exported file is in XML, so can be modified. You can import an
```
Comment (vbotnari12) [inline modules/ROOT/pages/configure-a-test-organization.adoc]: The exported file is in XML,  => The exported file is in XML format,

File: modules/ROOT/pages/configure-a-test-organization.adoc [hunk 16]
```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
...
-organization definition in this XML format that was exported from another Bonita Studio running the same version of Bonita, or that was constructed by some other method but has the same format. You can also import the organization file into Bonita Portal, ready for production.
+organization definition in this XML format that was exported from another Bonita Studio running the same version of Bonita, or that was constructed by some other method but has the same format. You can also import the organization file into Bonita Administrator Application, ready for production.
```
Comment (vbotnari12) [inline modules/ROOT/pages/configure-a-test-organization.adoc:50]: in this XML format that was exported from another Bonita Studio => in the XML file that was exported from another Bonita Studio

File: modules/ROOT/pages/configure-a-test-organization.adoc (hunk 2 above)
Comment (vbotnari12) [reply modules/ROOT/pages/configure-a-test-organization.adoc]: I dont see it in this PR. is it in the third PR?

File: modules/ROOT/pages/engine-architecture-overview.adoc (hunk 12 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/engine-architecture-overview.adoc]: Added it

File: modules/ROOT/pages/execution-sequence-states-and-transactions.adoc (hunk 13 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/execution-sequence-states-and-transactions.adoc]: Added

File: modules/ROOT/pages/configurable-archive.adoc (hunk 14 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configurable-archive.adoc]: Done

File: modules/ROOT/pages/configure-a-test-organization.adoc (hunk 15 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configure-a-test-organization.adoc]: Added

File: modules/ROOT/pages/configure-a-test-organization.adoc (hunk 16 above)
Comment (DumitruCorini) [reply modules/ROOT/pages/configure-a-test-organization.adoc:50]: Added

**Review by vbotnari12 — APPROVED** [review]
all conversations are resolved.

**Pull request merged**